source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/frame_transformer/frame_transformer.py

```diff
@@ -206,29 +201,42 @@ def _initialize_impl(self):
             )
 
         # The names of bodies that RigidPrimView will be tracking to later extract transforms from
-        tracked_body_names = list(body_names_to_frames.keys())
-        # Construct regex expression for the body names
-        body_names_regex = r"(" + "|".join(tracked_body_names) + r")"
-        body_names_regex = f"{self.cfg.prim_path.rsplit('/', 1)[0]}/{body_names_regex}"
+        tracked_prim_paths = [body_names_to_frames[body_name]["prim_path"] for body_name in body_names_to_frames.keys()]
+        tracked_body_names = [body_name for body_name in body_names_to_frames.keys()]
+
+        body_names_regex = [tracked_prim_path.replace("env_0", "env_*") for tracked_prim_path in tracked_prim_paths]
+
         # Create simulation view
         self._physics_sim_view = physx.create_simulation_view(self._backend)
         self._physics_sim_view.set_subspace_roots("/")
         # Create a prim view for all frames and initialize it
         # order of transforms coming out of view will be source frame followed by target frame(s)
-        self._frame_physx_view = self._physics_sim_view.create_rigid_body_view(body_names_regex.replace(".*", "*"))
+        # self._frame_physx_view = self._physics_sim_view.create_rigid_body_view('/World/envs/env_*/{Robot/base,Robot/LH_SHANK,Robot/LF_SHANK,cube}')
+        self._frame_physx_view = self._physics_sim_view.create_rigid_body_view(body_names_regex)
```

Comment:
This operation for replace should still happen right?

```suggestion
        self._frame_physx_view = self._physics_sim_view.create_rigid_body_view(body_names_regex.replace(".*", "*"))
```

Reply:
No, this is no longer needed